test/e2e/specs/site-editor/styles.spec.js

```diff
@@ -54,6 +54,6 @@ test.describe( 'Styles', () => {
 		await paddingControl.fill( '2' );
 
 		// Check the padding value
-		await expect( block ).toHaveCSS( 'padding-left', '35.4644px' );
+		await expect( block ).toHaveCSS( 'padding-left', '35.4932px' );
```

Comment:
For some reason this line was failing:

<img width="766" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/935d29bc-4f7b-4aa8-af57-c74672bda5f8">

So I've updated this to the new expected value to see if it gets the tests passing. Separately, I wonder if this check should be simplified to ignore the fraction, as this effectively averages out to around 36px anyway 🤔

Reply:
Oh that's going to be a fun test to maintain 🤦

Reply:
I think it would be better to use a custom value - e.g. set it to 5px, and then the expected padding-left should be 5px. I can look at updating this as part of the tests I'm working on.

Reply:
Looks like it was just updated to a hard-coded value in https://github.com/WordPress/gutenberg/pull/62111 so I've reverted the change here.